### DISCUSSION TITLE
Stylizing thoughts of a character without changing a Theme two times for one sentence

### DISCUSSION THREAD

**Discussion by Ingeniouz** [dialogic-godot/dialogic#1141] [Ideas]
Hello, I just joined the community and don't know if someone thought of it already. I want to show my character's thoughts all the time, with italic font. The way I found to do it is this in Timeline:
1. Change Theme (to the one with italic font);
2. Text (with thought);
3. Change Theme (back).

If I'd want to make a scene where MC is thinking and talking, it would be hell.
Maybe there is a way I don't know?

Also I can't find the search feature in docs, maybe the answer is in FAQ or something but I didn't click at the right link.

**Comment by Ingeniouz** [comment]
Hey, the answers come right after I post stuff here, sorry.
I just found I can create a double for character, and select Thoughts Theme for him. Call him Character(thoughts) and set his Display Name as Character, and it works fine for me.

**Comment by Jowan-Spooner** [comment]
If it's just italic font you want, I believe you can use the bbcode [i] at the beginning of a text event to make it italic

**Reply by Ingeniouz** [reply]
Thanks, I didn't know bbcodes are working here. But it doesn't work with my font, [i] selected text [/i] is disappearing. Italic is another .ttf file, I guess this is the reason. The main question was solved, but I wonder if [i] can work.
